docs/user/quick-start.md

```diff
@@ -0,0 +1,80 @@
+We recommend deploying Kyma Lifecycle Manager with the control plane kustomize profile, and `kcp-system`, `kyma-system` namespace must be configured upfront.
+```
+kubectl create ns kcp-system
+kubectl create ns kyma-system
+kyma alpha deploy -c alpha -k https://github.com/kyma-project/lifecycle-manager/config/control-plane
+```
+
+If the deployment is successful, you should see the following main resources:
+
+- Kyma Lifecycle Manager deployment instance named as `klm-controller-manager` under `kcp-system` namespace
+- All Module Templates configured in [kyma official repository](https://github.com/kyma-project/kyma/tree/main/modules) get deployed in `control-plane` as target
+- A Kyma CR that uses the global `alpha` channel but without any module configured, sync disabled, named `default-kyma` under `kyma-system` namespace
+
+### Manage modules in single cluster mode
+By default, all Module Templates deployed by Kyma CLI will be configured for single cluster mode (`spec.target=control-plane`), which means the module will be deployed in the Control Plane cluster, together with Kyma Lifecycle Manager.
+
+With Kyma CLI [enable module](https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma_alpha_enable.md) and [disable module](https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma_alpha_disable.md) command, you can manage kyma modules without directly modify Kyma CR.
+
+As an example, you can use following command to enable keda module in `alpha` channel for `default-kyma` under `kyma-system` namespace
+```
+kyma alpha enable module keda -c alpha -n kyma-system -k default-kyma
+```
+### Manage modules in remote cluster mode
+To allow Kyma Lifecycle Manager manages Kyma modules in remote cluster, two prerequisite must be fulfilled.
+
+1. For the module to be deployed remotely, the related Module Template must be configured as `remote` in `spec.target`.
+2. A kubernetes secret resource which contains remote cluster kubeconfig access data must be deployed into control plane cluster upfront.
+
+The Module Template persists in [kyma official repository](https://github.com/kyma-project/kyma/tree/main/modules) are configured as `remote` by default, you can use following command deploy them:
+```
+kubectl apply -k https://github.com/kyma-project/kyma//modules
+```
+
+With the following command, it will create a secret yaml file which named `default-kyma` (same as the Kyma CR name) under `kyma-system` (same as the Kyma CR namespace) namespace, which contains remote cluster kubeconfig as `data.config`.
+```
+export KUBECONFIG=[path to your remote cluster kubeconfig yaml file]
+./hack/k3d-secret-gen.sh default-kyma kyma-system
+```
+Deploy this secret under local control plane cluster:
+```
+kubectl config use-context k3d-kyma 
+kubectl apply -f default-kyma-secret.yaml
+```
+
+After this access secrete successfully deployed, you can start to use Kyma CLI manage modules for remote cluster.
+
+## Next Steps
+[TBD]
```

Comment:
What is TBD here?

Reply:
I plan to put here some reference links to other user documents, but they are not created yet.

Reply:
Maybe write that in the TBD if you plan to commit it

Reply:
I remove this part, will add it in later refactoring